src/bars/index.tsx

```diff
@@ -10,6 +10,7 @@ import {BarsCanvas} from './styled';
 export interface AudioBarsProps {
   analyser: Analyser;
   dimensions?: Dimensions;
+  barWidth?: number;
```

Comment:
Do you think consumers would be more interested in specifying the number of bars? maybe we should offer both?